apollo-router/src/plugins/connectors/testdata/interface-object.yaml

```
@@ -0,0 +1,48 @@
+# rover supergraph compose --config apollo-router/src/plugins/connectors/testdata/interface-object.yaml > apollo-router/src/plugins/connectors/testdata/interface-object.graphql
+# the result is manually edited because 2.10 disallows interface object
+federation_version: =2.10.0-alpha.1
```

```suggestion
federation_version: =2.10.0-alpha.2
```

Set to the next release version, though if we don't get this change in before tomorrow, we'll want to bump to `.3` probably just to set future testers' expectations